Domain/Entities/Tenant.cs

```diff
@@ -0,0 +1,11 @@
+﻿using Domain.Common;
+
+namespace Domain.Entities;
+
+public class Tenant : BaseEntity
+{
+    public string? Name { get; set; }
+    public DateTime ActiveFrom { get; set; }
+    public DateTime ActiveTo { get; set; }
+    public Order Order { get; set; }
```

Comment:
kadangi tenant galima pasiimt is employee orderyje, tai sito nereikia irgi

Reply:
pataisyta